examples/flow_benchmark_3d_case_3.py

```diff
@@ -98,7 +99,7 @@ def intersection_permeability(self, subdomains: list[pp.Grid]) -> pp.ad.Operator
         return self.isotropic_second_order_tensor(subdomains, permeability)
 
 
-class FlowBenchmark3dCase3BoundaryConditions:
+class BoundaryConditions:
```

Comment:
Now that I saw you included the scaling of the influx in the Geiger 2d, I remember I haven't done that for this example. Maybe we want to do it for consistency? The change should only affect L133 of this module.

Reply:
I was thinking more about consistency in physical scales (`m` and `s`), in case someone would like to change the units that is.